background/redirects.rst

```diff
@@ -0,0 +1,88 @@
+.. _redirects:
+
+Redirects in Divio Cloud projects
+=================================
+
+You will find that various redirects at work on URLs in Divio Cloud projects.
+
+They fall into different categories:
+
+* protocol redirects (e.g. ``http://example.com`` to ``https://example.com``)
+* domain name redirects (e.g. ``https://www.example.com`` to ``https://example.com``)
+* language redirects (e.g. ``https://example.com/about`` to ``https://example.com/en/about``)
+
+
+Protocol redirects
+------------------
+
+Our projects are HTTPS-ready by default, and `we provide free SSL certificates even on free
+projects <http://support.divio.com/control-panel/projects/ssl-certificates-and-https-on-divio-cloud-
+projects>`_.
+
+Adding an environment variable ``SECURE_SSL_REDIRECT`` (set to ``True``) will enable these
+redirects.
+
+`SECURE_SSL_REDIRECT <https://docs.djangoproject.com/en/1.10/ref/settings/#secure-ssl-redirect>`_
+is handled by the Django's ``SecurityMiddleware``.
+
+By default, this is a ``302 Temporary Redirect``
+
+
+Domain name redirects
+---------------------
+
+You can set up your site's domains using the `Domains section of the Control Panel
+<http://support.divio.com/control-panel/projects/using-your-own-domain-with-divio-cloud>`_. This
+includes the ability to set a primary and secondary domains. The secondary domains domains will
+redirect to the primary domain.
+
+By default, this is a ``302 Temporary Redirect``
+
+
+Language redirects
+------------------
+
+Django provides redirection to the the default language URL when none is specified. (In addition,
+django CMS offers complex fallback options for unavailable languages.)
+
+For example, ``/about`` will redirect to ``/en/about`` if English is the default language.
+
+:ref:`Aldryn Django <aldryn-django>` can be configured to not use the prefix for the default language, by checking
+the *Remove URL language prefix for default language* checkbox on the settings for the Aldryn
+Django addon.
+
+Alternatively, you can set ``PREFIX_DEFAULT_LANGUAGE`` to false in ``settings.py``.
+
+Note that prior to version 1.10, this will not work with projects in which multiple languages
+are defined.
+
+This is a ``302 Temporary Redirect``
+
+
+301 Permanent and 302 Temporary redirects
+-----------------------------------------
+
+You will sometimes see online site-checking tools, such as moz.com, encouraging the use of
```

Comment:
Let's not mention ``moz.com`` by name here. This is like singling them out. There are many similar services, including ``moz.com``, doing a great job at finding SEO/Performance issues. The results are just not something to blindly follow under all circumstances.